### PR TITLE
:bug: :mortar_board: Topic15 Fix ordering of sections and add testing section

### PR DESCRIPTION
**Pull request by jameshughes89**
### What
Fix the order of the sections for Topic15. 
Add a short section saying to check out the tests on their own. 

### Why
Section order got messed up with merges. 
Tests are not directly discussed since they should be familiar with them by now, but still presented since all the test methods are far more comprehensive than previous.

### CODE REVIEW

**Review by twentylemon — COMMENTED** [review]
Diff is hard to read while sitting on the 🚽  -- any changes other than the testing section? Or all just re-ordering?

**Comment by jameshughes89** [comment]
> Diff is hard to read while sitting on the  -- any changes other than the testing section? Or all just re-ordering?

Just the testing section and reordering. The diff is super hard to follow on this one. Going forward, I think adding the main sections into the topics will help a lot.

**Review by twentylemon — APPROVED** [review]
